### ISSUE TITLE
Consider redundancy when checking for focusable descendants of aria-hidden elements

### ISSUE THREAD

**Issue by fkohrt** [GoogleChrome/lighthouse#11999]
**Feature request summary**

Lighthouse should take into account if an image's `aria-hidden` `figcaption` is identical with the image's `alt` text. Only if not identical, [[aria-hidden="true"] elements contain focusable descendants](https://web.dev/aria-hidden-focus/) should become a potential issue.

**What is the motivation or use case for changing this?**

[Pandoc](https://pandoc.org/) converts...

```md
![description](example.png)
```

...to...

```html
<figure>
<img src="example.png" alt="description" /><figcaption aria-hidden="true">description</figcaption>
</figure>
```

`description` is already available as `alt`, therefore the figcaption is being hidden from screen readers to avoid redundancy (see jgm/pandoc#6495 for reasoning).

**How is this beneficial to Lighthouse?**

Improves accuracy of accessibility assessment.

**Comment by patrickhulce** [comment]
Thanks very much for filing @fkohrt! The accessibility audits are implemented with [axe](https://github.com/dequelabs/axe-core), and we'll respect whatever is decided there. Any future updates there will be picked up in a future Lighthouse update if you'd be interested in filing there :)

**Comment by fkohrt** [comment]
@patrickhulce I see, didn't know that. Thanks for redirecting me, this is now dequelabs/axe-core#2780
